R/pkg/DESCRIPTION

```diff
@@ -54,5 +54,5 @@ Collate:
     'types.R'
     'utils.R'
     'window.R'
-RoxygenNote: 5.0.1
+RoxygenNote: 6.0.1
```

Comment:
perhaps we don't want to change this - this new version of Roxygen is not on Jenkins